devHelper/README.md

```diff
@@ -12,6 +12,10 @@ In the root directory of this repo, run the following command:
 ```
 sh ./generate_data.sh
 ```
+On Ubuntu may need to run directly
+```
+./generate_data.sh
```

Comment:
i think in nowadays this is a more regular way to run script files, so maybe just replace `sh ./generate_data.sh` with this

Reply:
@ocshawn still want to update this one?

Reply:
yes would still like to update this

Reply:
sorry just saw your note, i dont run things on mac so i dont know if it will work without the sh

Reply:
yeah it can run on recent MacOSs so we are safe to update it